tests/docker-build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Exit on error
+set -e
+# Print executed commands
+set -x
+
+pushd ..
```

Comment:
So it goes to `/usr/local/`?

Reply:
Yes, maybe we can create subdirectory for building the package